docs/source/contributor-guide/benchmarking_macos.md

```diff
@@ -61,6 +61,16 @@ Start Spark in standalone mode:
 $SPARK_HOME/sbin/start-master.sh
 ```
 
+Note: if there is no `$SPARK_HOME/sbin/start-master.sh` on your machine it is possible to start master by running 
```

Comment:
Just curious, why would this file ever be missing since it is included in the distribution tarball?

Reply:
Came to that, because the file missing on my local machine with Spark installed through `brew` 🤷